src/Microsoft.AspNet.Mvc.Core/ActionResults/JsonResult.cs

```diff
@@ -39,6 +40,16 @@ public JsonResult(object value)
         /// Creates a new <see cref="JsonResult"/> with the given <paramref name="data"/>.
         /// </summary>
         /// <param name="value">The value to format as JSON.</param>
+        /// <param name="serializerSettings">The <see cref="JsonSerializerSettings"/> to be used by the formatter.</param>
+        public JsonResult(object value, JsonSerializerSettings serializerSettings)
```

Comment:
`NotNull` for the serializersettings parameter (could you make similar change to the other overload taking in the formatter)